### PR TITLE
feat: add Rick apps

### PR DESCRIPTION
**Pull request by tumido**
Part of: https://github.com/operate-first/hetzner-baremetal-openshift/issues/12
Resolves: https://github.com/operate-first/hetzner-baremetal-openshift/issues/12
Requires: https://github.com/operate-first/apps/pull/851 https://github.com/operate-first/apps/pull/852 https://github.com/operate-first/apps/pull/853 https://github.com/operate-first/apps/pull/854

All apps (except app of apps) have autosync disabled for now.

### CODE REVIEW

**Comment by tumido** [comment]
/hold

for changes in cluster resources

File: envs/emea/rick/cluster-management/alertreceiver.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+kind: Application
+metadata:
+  name: alertreceiver
+spec:
+  destination:
+    name: rick
+    namespace: opf-alertreceiver
+  project: cluster-management
+  source:
+    path: alertreceiver/overlays/moc/rick
```
Comment (HumairAK) [inline envs/emea/rick/cluster-management/alertreceiver.yaml:11]: moc > emea

Comment (HumairAK) [reply envs/emea/rick/cluster-management/alertreceiver.yaml:11]: same applies to the other paths

Comment (tumido) [reply envs/emea/rick/cluster-management/alertreceiver.yaml]: fixed :+1:

**Comment by tumido** [comment]
/cc @HumairAK 

please review once again. :slightly_smiling_face:

**Comment by tumido** [comment]
once merged resolves https://github.com/operate-first/hetzner-baremetal-openshift/issues/12

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**